src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -139,14 +145,14 @@ IF NOT EXISTS (SELECT * FROM sys.external_data_sources WHERE name = '<dataSource
               LOCATION = '<destinationParquetFile>',
               DATA_SOURCE = [<destinationDataSourceName>], /* metadata container */
               FILE_FORMAT = [<fileFormat>]
-          ) AS SELECT    datarepo_row_id,
+          ) AS SELECT
```

Comment:
I think we still want `datarepo_row_id` no?

Reply:
I'm now adding it in the list of columns on L1130 because I also need to define the schema for in the WITH() clause. By including it in the list of columns, it automatically gets added in both spots.  An alternate would have been to also manually also added in the WITH() clause.